frontend/src/controllers/bookingController.js

```diff
@@ -0,0 +1,45 @@
+app.controller('bookingController', ['$scope', '$http','$routeParams', '$log','$location',
+    function ($scope, $http, $routeParams, $log,$location) {
+
+        $scope.movieId = $routeParams.movieId;
+        
+        $http.get(`http://localhost:3000/movies?id=${$scope.movieId}`)
```

Comment:
We don't need an extra call to the backend, we can store the data we need when clicking the movie details